<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1503</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="drawing_digit" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_fc_in" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>660</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_conv_input" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>91</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_conv_weight" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_conv_out" native="true">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>671</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_fc_weight" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>201</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_fc_out" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>671</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Draw a number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Input Vector&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Output Feature Vectors&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Fully Connected Layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>470</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Classification Result&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Convolutional Layer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_read">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>830</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Read conductance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_classify">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Classify with Analog DPE Engine</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="mpl_digit" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
